lib/erb_lint/cli.rb

```diff
@@ -41,7 +41,7 @@ def run(args = ARGV)
       if !@files.empty? && lint_files.empty?
```

Comment:
Something for a follow-up PR, but thought I'd call it out here. We reference `lint_files` a bunch in this function, but the data isn't memoized/stored anywhere, so it gets recomputed a lot. For a large codebase, this could be a significant cost.

Reply:
Good find. I can make another PR to store `lint_files` somewhere so it doesn't get recomputed each time.

Reply:
https://github.com/Shopify/erb-lint/pull/149